app/src/main/res/layout/item_contributor_header.xml

```diff
@@ -17,10 +17,18 @@
             android:id="@+id/contributor_header_text"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
+
+            android:layout_marginEnd="72dp"
             android:layout_marginStart="72dp"
             android:layout_marginTop="36dp"
+            android:layout_marginBottom="36dp"
+
```

Comment:
Can you remove the extra line space? 🙏

Reply:
Okay, I'll remove the line. include L24 or not?

Reply:
no no. I mean L20, L25 and L30.

Reply:
Okay, It means "Fill the gap" right? Now, I'll delete a gap. plz wait...